MediaBrowser.Model/Querying/ItemFields.cs

```diff
@@ -209,6 +209,7 @@ public enum ItemFields
         ExternalEtag,
         PresentationUniqueKey,
         InheritedParentalRatingValue,
+        InheritedParentalRatingSubValue,
```

Comment:
IMO we should always set this if the InheritedParentalRatingValue field is requested instead of separating both.

Reply:
#13818 removed all unsupported fields